gpkit/constraints/signomial_program.py

```diff
@@ -126,6 +127,11 @@ def localsolve(self, solver=None, verbosity=1, x0=None, rel_tol=1e-4,
         self.sens_from_gpconstr(gp.constraints,
                                 result["sensitivities"]["constraints"],
                                 result["sensitivities"]["constants"])
+
+        for constraint in self.constraints:
```

Comment:
`process_result` is inherited and so you should just be able to call `self.process_result(result)`